papaparse.js

```diff
@@ -519,8 +523,21 @@ License: MIT
 
 			// Rejoin the line we likely just split in two by chunking the file
 			var aggregate = this._partialLine + chunk;
+			this._skipLines = parseInt(this._config.skipFirstNLines) || 0;
+			this._pendingSkip = parseInt(this._config.skipFirstNLines) || 0;
+			this._skipHeader = this._config.header ? 1 : 0;
```

Comment:
Wont' be simpler to just increse lines to skip by one when we have a header?

Reply:
This has been fixed